src/InputWithLabel.jsx

```diff
@@ -0,0 +1,21 @@
+import { useRef } from "react";
+
+function InputWithLabel(props) {
+  const inputRef = useRef(null);
+
+  return (
+    <>
+      <label htmlFor="todoTitle">{props.label}</label>
+      <input
+        id="todoTitle"
+        name="title"
+        value={props.todoTitle}
+        onChange={props.onTitleChange}
+        autoFocus
```

Comment:
Per the assignment instructions, you'll need to replace this `autoFocus` prop with refocusing the input element using a `useEffect` in this component.